src/parser/smt2/Smt2.g

```diff
@@ -1788,6 +1796,20 @@ symbolicExpr[CVC4::SExpr& sexpr]
  */
 term[CVC4::Expr& expr, CVC4::Expr& expr2]
 @init {
+  std::string name;
+}
+: termNonVariable[expr, expr2]
+    /* a variable */
+  | symbol[name,CHECK_DECLARED,SYM_VARIABLE]
+    { expr = PARSER_STATE->getExpressionForName(name); }
```

Comment:
What happen if expr is NULL here?

Reply:
Expression can't be null here.  I noticed that assert(...)'s are not too common in this file I suppose since CVC4 assertions aren't available, that's why I typically haven't been adding more assertions to ANTLR files.  Regardless, I'd prefer them so I'll add assert( !expr.isNull() );